pkgs/by-name/cr/cross-seed/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+
+buildNpmPackage rec {
+  pname = "cross-seed";
+  version = "6.0.0-43";
+
+  src = fetchFromGitHub {
+    owner = "cross-seed";
+    repo = "cross-seed";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```

Might be a matter of preference though

---

`tag ="v${version}"` is the preferred option since recently.